apps/console/src/features/claims/components/add/add-external-claim.tsx

```diff
@@ -261,6 +261,14 @@ export const AddExternalClaims: FunctionComponent<AddExternalClaimsPropsInterfac
                                             { type: resolveType(attributeType) }));
                                     }
                                 }
+                                if (attributeType === ClaimManagementConstants.SCIM) {
+                                    if (!value.toString().match(/^[a-zA-Z]([-/_/\w]+)*$/)) {
```

Comment:
This expression matches forward slashes as well `/` and `_` is obsolete when used with `\w`. We don't need to wrap the rest with a capture group with zero or many quantifiers. It's unnecessary.

To match a string that must start with alphabetic char and rest with `-`, `_`, `0-9`, `a-z`, `A-Z` then the following expression will do the validation `^[a-zA-Z][-\w]+$`

Reply:
Also consider using `test` for evaluating strings instead of `match` i.e., `/^[a-zA-Z][-\w]+$/.test("SampleString")`